src/raftstore/store/snap/io.rs

```diff
@@ -137,9 +136,14 @@ pub fn apply_sst_cf_file(path: &str, db: &DB, cf: &str) -> Result<(), Error> {
     Ok(())
 }
 
-fn create_sst_file_writer(snap: &DbSnapshot, cf: CfName, path: &str) -> Result<SstWriter, Error> {
+fn create_sst_file_writer(
+    snap: &DbSnapshot,
+    cf: CfName,
+    path: &str,
+) -> Result<RocksSstWriter, Error> {
```

Comment:
here we still use Rocks? will we return a SstWrtier interface later?

Reply:
Yes, later the abstraction will be pushed further out.